streampipes-extensions/streampipes-processors-enricher-jvm/src/main/java/org/apache/streampipes/processors/enricher/jvm/processor/limitsenrichment/QualityControlLimitsEnrichmentProcessor.java

```diff
@@ -36,7 +36,7 @@
 import org.apache.streampipes.wrapper.params.compat.ProcessorParams;
 import org.apache.streampipes.wrapper.standalone.StreamPipesDataProcessor;
 
-public class QualityControlLimitsEnrichment extends StreamPipesDataProcessor {
+public class QualityControlLimitsEnrichmentProcessor extends StreamPipesDataProcessor {
```

Comment:
Can we use the `IStreamPipesDataProcessor` interface instead of extending `StreamPipesDataProcessor`?
https://streampipes.apache.org/docs/extend-tutorial-data-processors/

Reply:
Thanks, I changed it. Is there a reason why we did not deprecate `StreamPipesDataProcessor`?

Reply:
Probably because we didn't want to refactor all existing pipeline elements ;-)